v19.2/cockroachcloud-upgrade-policy.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cluster Upgrades
+summary: Learn about the CockroachCloud upgrade policy.
+toc: true
+build_for: [cockroachcloud]
+---
+
+This page describes the upgrade policy for CockroachCloud.
+
+CockroachCloud supports the latest major version of CockroachDB and the version immediately preceding it. Support for these versions includes minor version updates and security patches.
+
+## Minor Version Upgrades
+"Minor" versions (or "point" releases) are stable, backward-compatible improvements to the major versions of CockroachDB.
+
+CockroachCloud automatically upgrades all clusters to the latest supported minor version (for example, v19.2.1 → v19.2.2) within the 2 business days after a minor version release. Starting April, the [CockroachCloud Admin(s)](cockroachcloud-console-access-management.html#console-admin) will be notified before and after their clusters are upgraded to the latest minor version.
```

Comment:
@joshimhoff Should we remove this para? ("CC automatically upgrades...minor version). 
@emily-horing Or would you prefer we increase the cap (from 2 days to 4 days) but retain the para?

Reply:
Can we just say "CockroachCloud upgrades all clusters to the latest supported minor version" and leave it at that? That way customers know that they are always getting point releases but we don't have to commit to anything yet.

Reply:
This SGTM. Thanks!

Reply:
IMO, that would cause more confusion. Like, it can set the expectation that we upgrade the clusters as soon as the version is released. I would rather give them a timeframe or not mention it at all. However, I might be getting paranoid here. I am okay with what y'all decide.

Reply:
Okay..going with Emily's suggestion.

Reply:
The minor upgrade automation will be the first to be completed (AFAIK) so this will also be the first thing we can update, which makes me more comfortable letting it be vague for now. Thanks Amruta!!